packages/daimo-expo-app-delegate/ios/DaimoAppDelegate.swift

```diff
@@ -18,8 +18,8 @@ public class DaimoAppDelegate: ExpoAppDelegateSubscriber {
     
     if self.isOnMac() {
       UIApplication.shared.connectedScenes.compactMap { $0 as? UIWindowScene }.forEach { windowScene in
-        windowScene.sizeRestrictions?.minimumSize = CGSize(width: 500, height: 900)
-        windowScene.sizeRestrictions?.maximumSize = CGSize(width: 500, height: 900)
+        windowScene.sizeRestrictions?.minimumSize = CGSize(width: 500, height: 1000)
```

Comment:
telegram is smaller than 900 even, but lgtm

Reply:
fixes a bug where the cancel payment link button was cut off

<img width="385" alt="image" src="https://github.com/daimo-eth/daimo/assets/169280/195be0ff-420e-47d4-853b-443a408d7161">

Reply:
I don’t think this fixes that? That happens on mobile too, probably need to change the 450s here: https://github.com/daimo-eth/daimo/blob/955e25bb2844e923b2d166bd8d23af1107d2dc98/apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx#L72

Reply:
ah i thought it was a percentage. fixed

Reply:
<img width="250" alt="image" src="https://github.com/daimo-eth/daimo/assets/169280/814b39ea-67f4-48ed-ac75-018bc366fb77">